dev/buildtool/spin_commands.py

```diff
@@ -129,13 +129,12 @@ def build_all_distributions(self, repository):
       labels = {'repository': repository.name,
                 'dist': dist_arch.dist,
                 'arch': dist_arch.arch}
-      cmd = 'go get -v -u {}'.format(spin_package_path)
```

Comment:
For future reference, @ttomsu and I determined that the `-u` flag would indicate to `go get` that it should update pre-installed dependencies. In our case, there are no pre-installed dependencies on each clean build. Regardless, it is worth removing to make this command clearer.

Reply:
Thanks for the explanation, I was wondering about that!